x-pack/apm-server/aggregation/txmetrics/aggregator.go

```diff
@@ -594,3 +644,35 @@ func transactionCount(tx *model.Transaction) float64 {
 	}
 	return 1
 }
+
+func writeLabels(w io.StringWriter, l model.Labels) {
+	if len(l) == 0 {
+		return
+	}
+	for key, value := range l {
```

Comment:
Labels is a map, so iteration will be non-deterministic. Should we iterate over the sorted keys?

Perhaps when copying the global labels in makeTransactionAggregationKey, create a slice and sort that?